content/post/100-awesome-women-in-the-open-source-community-you-should-know.md

```diff
@@ -3,7 +3,6 @@ author: eiso
 date: 2016-05-25 
 title: "100 Awesome Women In The Open-Source Community You Should Know" 
 draft: false 
-image: /post/100-awesome-women-in-the-open-source-community-you-should-know/intro.jpg 
```

Comment:
available images for that post:
![image](https://user-images.githubusercontent.com/2437584/27126682-9725c0fa-50f8-11e7-9aed-f969ee4f8ffa.png)